src/components/data/PathListAutomation.js

```diff
@@ -193,246 +200,293 @@ export default function PathListAutomation(props) {
-                                style={{
-                                    position: "absolute",
-                                    top: "50%",
-                                    left: "50%",
-                                }}
-                            />
-                        )}
-
-                        <Grid
-                            container
-                            direction="column"
-                            justify="center"
-                            alignItems="stretch"
-                            spacing={1}
-                        >
-                            {createPathListError && (
-                                <Grid item xs>
-                                    <ErrorTypographyWithDialog
-                                        baseId={baseId}
-                                        errorMessage={errorMsg}
-                                        errorObject={createPathListError}
-                                    />
-                                </Grid>
-                            )}
-                            <Grid item xs>
-                                <Typography variant="body2">
-                                    {t("pathListInputLbl")}
-                                </Typography>
-                                {infoTypeError && (
-                                    <ErrorTypographyWithDialog
-                                        baseId={baseId}
-                                        errorMessage={t("infoTypeFetchError")}
-                                        errorObject={infoTypeError}
-                                    />
-                                )}
-                                <Field
-                                    id={buildId(
-                                        baseId,
-                                        ids.PATH_LIST_AUTO_INPUTS
-                                    )}
-                                    name="selectedPaths"
-                                    required={true}
-                                    component={MultiInputSelector}
-                                    height="25vh"
-                                    label={t("suggestionSelection_any_plural")}
-                                />
-                            </Grid>
-                            <Grid item xs>
-                                <Field
-                                    id={buildId(
-                                        baseId,
-                                        ids.PATH_LIST_AUTO_FOLDERS_ONLY_SWITCH
+        <>
+            <Formik
+                onSubmit={handlePathListCreation}
+                validate={validate}
+                initialValues={{
+                    selectedPaths: [],
+                    dest: startingPath,
+                    fileName: "",
+                    pattern: "",
+                    foldersOnly: false,
+                }}
+            >
+                {({ handleSubmit }) => {
```

Comment:
To get the validations to run, you'll need to use this `handleSubmit` function (hopefully your IDE was showing you that this was not being used or called anywhere).  You'll note on your `Submit` button, there's no `onClick` defined.  You want this function to run when the `Submit` button is clicked